behat.yml.dist

```diff
@@ -397,6 +397,16 @@ default:
             filters:
                 tags: "@security"
 
+        ui:
```

Comment:
Or maybe: ``ui-checkout``, for easier sorting of ``ui-``, ``domain-`` and ``api-`` suites.